uber/models/__init__.py

```diff
@@ -1128,7 +1128,8 @@ def search(self, text, *filters):
                     last, first = first.strip(','), last
                 name_cond = attendees.icontains_condition(first_name=first, last_name=last)
                 legal_name_cond = attendees.icontains_condition(legal_name="{}%{}".format(first, last))
-                return attendees.filter(or_(name_cond, legal_name_cond))
+                if attendees.filter(or_(name_cond, legal_name_cond)).first():
```

Comment:
This actually partly fixes https://github.com/magfest/ubersystem/issues/3094 because I ran into it while testing the promo code group name search and I thought it was dumb.

Reply:
This seems like a good compromise solution.